highfive/newpr.py

```diff
@@ -156,6 +164,8 @@ def is_collaborator(self, commenter, owner, repo):
         except urllib2.HTTPError, e:
             if e.code == 404:
                 return False
```

Comment:
If we use `retry` here, it will change the behavior of this method when all of the attempts fail. It would be nice if `retry` could return a value we provide on the final failure.